cmd/prometheus/main.go

```diff
@@ -259,6 +262,31 @@ func reloadConfig(filename string, rls ...Reloadable) (err error) {
 		return fmt.Errorf("couldn't load configuration (-config.file=%s): %v", filename, err)
 	}
 
+	// Add AlertmanagerConfigs for legacy Alertmanager URL flags.
```

Comment:
If one is deprecated and one experimental, there's no obviously correct path for the user.
We should decide for either.